biztown/src/components/TableRow.js

```diff
@@ -1,96 +1,89 @@
-import React from "react"
```

Comment:
small stylistic nit: could we center align the radio buttons in the column

Reply:
Added to backlog.